chore: Translate all documentation to English (#1054)
Amsterdam/design-system: storybook/storybook-docs/src/introduction.md

# Amsterdam Design System

> ## What is it

Reviewer: I think we can remove this, it's conventional that the first paragraph after the h1 explain what you're looking at

> The Amsterdam Design System is the library for the user interfaces (UI) of all our digital products and services.

Reviewer: ```suggestion
Amsterdam Design System is the library for the user interfaces (UI) of all our digital products and services.
```

This way, everyone works together faster and better, with more time to create actual value.
All software we create, purchase and use is easier to align with our corporate identity.
For Amsterdam residents, all our channels appear the same, work similarly, are widely accessible, and inspire trust.

## How does it work

The most concrete products of the Amsterdam Design System are the reusable user interface components, such as buttons, text components, menus, and form fields.
They adhere to the City’s branding: colour, typography and dimensions.
They adapt responsively to different screens and devices and are accessible to everyone.

We make these components available in various libraries so that designers and developers can use them in their work:

### Figma

[Find our Figma Library here](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?type=design&node-id=2927%3A29177&mode=design&t=6KlrHnKkHU2uZ9s9-1) (if necessary, create an account on your @amsterdam.nl address).
Various components and text styles are ready to use in your designs.
Several others are in the final stages of review.
Refer to the introduction and legend on the cover page.

> ### Reply

Reviewer: ```suggestion
### React
```

These components can be installed and used in your application [via npm](https://www.npmjs.com/search?q=%40amsterdam%2Fdesign-system).

### React Native

While developing the Amsterdam App, many UI components have been built according to the updated corporate identity.
They are still [in that repository](https://github.com/Amsterdam/amsterdam-app-frontend) but will eventually be transferred to the design system for other apps to use.

### CSS

The components are usable without React by applying the correct classes.
If you would like to follow this approach, please let us know.

### Salesforce

For the new amsterdam.nl, we provide a library of correctly designed Lightning Web Components usable in Salesforce CMS.

### Mendix

We help applications implemented in this low-code platform match the corporate identity.

## We build on the NL Design System

Amsterdam Design System uses the architecture of [NL Design System](https://nldesignsystem.nl/).

Thanks to the NL Design System, the entire government in the Netherlands can work together to provide understandable, user-friendly and accessible online services.

> A service that is logical and coherent but not necessarily uniform because, within the agreements, there is room for the identity of government organizations.

Reviewer: ```suggestion
A service that is logical and coherent but not necessarily uniform, because it allows room for the identity of government organizations.
```

> The choice to connect to this also makes it easier to offer different libraries.

Reviewer: ```suggestion
The choice to connect to NL Design System also makes it easier for us to create different libraries.
```

## Contact us

### Join Teams

> Do you work at or for the City of Amsterdam, [register now for the Design System team](https://teams.microsoft.com/l/team/19%3afYKS_RD2n1q4UhguA9jwEJk0A_VjYPO4TiLQjYlG_bo1%40thread.tacv2/conversations?groupId=381b5f11-b342-4a3a-8a78-8b371a90457d&tenantId=72fca1b1-2c2e-4376-a445-294d80196804) from MS Teams.

Reviewer: ```suggestion
If you work for the City of Amsterdam, [register now for the Design System team](https://teams.microsoft.com/l/team/19%3afYKS_RD2n1q4UhguA9jwEJk0A_VjYPO4TiLQjYlG_bo1%40thread.tacv2/conversations?groupId=381b5f11-b342-4a3a-8a78-8b371a90457d&tenantId=72fca1b1-2c2e-4376-a445-294d80196804) on MS Teams.
```

This way, you can easily stay informed, and we can keep in touch.

> Reading the channels sideways can help you, but you can also participate more.

Reviewer: ```suggestion
The channels have a lot of information you can read, and of course you are more than welcome to participate in threads.
```
